app/src/main/java/edu/grinnell/grinnell_publications_android/Models/Realm/RealmPublication.java

```diff
@@ -0,0 +1,66 @@
+package edu.grinnell.grinnell_publications_android.Models.Realm;
+
+import java.util.List;
+
+import edu.grinnell.grinnell_publications_android.Models.Interfaces.Publication;
+import edu.grinnell.grinnell_publications_android.Models.Interfaces.Series;
+import edu.grinnell.grinnell_publications_android.Models.Interfaces.Story;
+import io.realm.RealmList;
+import io.realm.RealmObject;
+
+/**
+ * This class implements @code{Publication} using Realm persistence.
+ *
+ * @author Albert Owusu-Asare
+ * @version 1.1 Fri May  6 02:40:59 CDT 2016
+ * @see RealmObject
+ * @see Publication
+ */
+public class RealmPublication extends RealmObject implements Publication {
+    private String publicationName;
+    private int publicationId;
+    private RealmList<RealmSeries> featuredSeries;
```

Comment:
Resolved - This was going off with Alex's original sketch. but noted 👍  getSeries(), getStories() etc is more concise.